app/pages/index.tsx

```diff
@@ -134,6 +151,61 @@ const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
 
 export default Home;
 
+export const DosesDescription = ({
```

Comment:
Copy from description list and refactor - this page will be removed eventually anyway 🙏